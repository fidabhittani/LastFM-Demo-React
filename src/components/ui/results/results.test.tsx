import React from "react";
import { shallow } from "enzyme";
import { EventsResultContainer, ArtistsResultContainer } from "./index";

/** Results TEST */
describe("Results Root Component Tests", () => {
  it("Results Component EventsResultContainer renders correctly", () => {
    const eventConntainerWrapper = shallow(<EventsResultContainer />);
    expect(eventConntainerWrapper).toMatchSnapshot();
  });

  it("Results Component ArtistsResultContainer renders correctly", () => {
    const artistsContainerWrapper = shallow(<ArtistsResultContainer />);
    expect(artistsContainerWrapper).toMatchSnapshot();
  });
});
