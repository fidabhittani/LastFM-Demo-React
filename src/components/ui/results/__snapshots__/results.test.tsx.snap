// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results Root Component Tests Results Component ArtistsResultContainer renders correctly 1`] = `ShallowWrapper {}`;

exports[`Results Root Component Tests Results Component EventsResultContainer renders correctly 1`] = `ShallowWrapper {}`;
